dts/bindings/sensor/ti,ina226.yaml

```diff
@@ -0,0 +1,93 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    TI INA226 Bidirectional Current and Power Monitor.
+    The <zephyr/dt-bindings/sensor/ina226.h> file should be included in the
+    DeviceTree and it provides macros that can be used for initializing the
+    configuration registers.
+
+    Example:
+      /* Board - mr_canhubk3. */
+      &lpi2c1 {
+        status = "okay";
+        pinctrl-0 = <&lpi2c1_default>;
+        pinctrl-names = "default";
+        clock-frequency = <I2C_BITRATE_STANDARD>;
+
+        ina226: ina226@41 {
+          compatible = "ti,ina226";
+          reg = <0x41>;
+          current-lsb-microamps = <5000>;
+          rshunt-micro-ohms = <500>;
+        };
+      };
+
+
+compatible: "ti,ina226"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  avg-count:
+    type: int
+    description: |
+      Number of samples to average (applies to all inputs).
+      Default is the power-on reset value.
+    default: 1
+    enum: [1, 4, 16, 64, 128, 256, 512, 1024]
+
+  vbus-conversion-time-us:
+    type: int
+    description: |
+      Vbus conversion time in microseconds.
+      Default is the power-on reset value.
+    default: 1100
+    enum: [140, 204, 332, 588, 1100, 2116, 4156, 8244]
+
+  vshunt-conversion-time-us:
+    type: int
+    description: |
+      Vshunt conversion time in microseconds.
+      Default is the power-on reset value.
+    default: 1100
+    enum: [140, 204, 332, 588, 1100, 2116, 4156, 8244]
+
+  operating-mode:
+    type: string
+    description: |
+      Selects mode of operation.
+      Default is the power-on reset value.
+    default: "Shunt and Bus, Continuous"
+    enum:
+      - "Power-Down (or Shutdown)"
+      - "Shunt Voltage, Triggered"
+      - "Bus Voltage, Triggered"
+      - "Shunt and Bus, Triggered"
+      - "Power-Down (or Shutdown)"
+      - "Shunt Voltage, Continuous"
+      - "Bus Voltage, Continuous"
+      - "Shunt and Bus, Continuous"
+
+  current-lsb-microamps:
+    type: int
+    required: true
+    description: |
+      This value should be selected so that measurement resolution is
```

Comment:
this description does not make clear what the value means

Reply:
Hi @decsny, I have addressed this in the latest push. Kindly check if the description is clear now. If not then please let me know. Thank you.